ui/imports/shared/views/PasswordView.qml

```diff
@@ -78,14 +76,26 @@ ColumnLayout {
         property bool containsNumbers: false
         property bool containsSymbols: false
 
-        readonly property var validator: RegExpValidator { regExp: /^[!-~]{0,64}$/ } // That incudes NOT extended ASCII printable characters less space and a maximum of 64 characters allowed
+        readonly property var validatorRegexp: /^[!-~]{0,64}$/
+        readonly property var validator: RegExpValidator { regExp: d.validatorRegexp } // That incudes NOT extended ASCII printable characters less space and a maximum of 64 characters allowed
+        readonly property string validatorErrMessage: qsTr("Only letters, numbers, underscores and hyphens allowed")
 
         // Password strength categorization / validation
         function lowerCaseValidator(text) { return (/[a-z]/.test(text)) }
         function upperCaseValidator(text) { return (/[A-Z]/.test(text)) }
         function numbersValidator(text) { return (/\d/.test(text)) }
         // That incudes NOT extended ASCII printable symbols less space:
         function symbolsValidator(text) { return (/[!-\/:-@[-`{-~]/.test(text)) }
+        function inputValidator(text) { return (validatorRegexp).test(text) }
```

Comment:
You don't seem to be using this function at all, except inside `validateInput(text)`; remove it?

Reply:
Also, the name suggests it returns some kind of validator when in fact in validates (tests) the input

Reply:
Done